starter/src/index.html

```diff
@@ -55,20 +15,16 @@
 	<!-- Webpack Dev Server reload -->
 	<script src="/webpack-dev-server.js"></script>
 	<% } %>
-
+	
+	<!-- Webpack DLL CSS files -->
```

Comment:
This comment was ok where it was before... just below this line: 
`<% if (htmlWebpackPlugin.options.dllFiles) { %>  <!-- dllFiles will be filled in by the Dll plugin -->`

Look at the one in the showcase...

Reply:
I'm so sorry, I have no idea why I had done that, it is so not logical ... must be really hot in my attic, I'm loosing my mind ^^'

Reply:
Hehehe no problem, it is Friday as well :p